tests/xlsx/__snapshots__/xlsx_export.test.ts.snap

```diff
@@ -35060,7 +35075,7 @@ exports[`Test XLSX export references with headers should be converted to referen
                         <!-- x-coordinate values -->
                         <c:numRef>
                             <c:f>
-                                Sheet1!B3:B100
+                                Sheet1!$B$3:$B$100
```

Comment:
why are those suddenly fixed ranges ? I don't think exporting a trendline should change that